### ISSUE TITLE
Pass an exclude to a linter package?

### ISSUE THREAD

**Issue by febbraro** [alecthomas/gometalinter#565]
The `gosec` package is flagging an error that I don't care to evaluate across my entire codebase.  

`gosec` has a way to pass an exclude for a particular rule

```
# Run everything except for rule G303
$ gosec -exclude=G303 ./...
```

Is there a way to pass something like that down to the `gosec` package?

**Comment by dnephin** [comment]
Yes, you can customize the command that is run from the config: 
```json
{
  "Linters": {
    "gosec": {"Command": "gosec -exclude=G303"}
  }
}
```

https://github.com/alecthomas/gometalinter#adding-custom-linters

**Comment by febbraro** [comment]
@dnephin Thanks so much for that pointer. Works like a charm!
